cmd/user/internal/application/eventhandler/when_user_access_token_was_requested.go

```diff
@@ -20,7 +20,7 @@ func WhenUserAccessTokenWasRequested(config oauth2.Config, secretKey string) eve
 
 		log.Printf("[EventHandler] %s\n", event.Payload)
 
-		e := user.WasRegisteredWithGoogle{}
+		e := user.WasRegisteredWithEmail{}
```

Comment:
Merge master and resolve conflicts (I merged https://github.com/vardius/go-api-boilerplate/pull/43)